src/pages/Homepage.js

```diff
@@ -10,12 +12,43 @@ class Homepage extends React.Component {
 
     this.state = {
       contacts: []
-    };
-  }
+      };
+
+    }
 
+    
   // invoked immediately after a component is mounted, good place for network requests
+<<<<<<< HEAD
```

Comment:
Can you resolve this merge conflict? It looks like you should keep HEAD and ditch the code from lines 49-63

Reply:
Okay I deleted those lines and re-pushed, should be fixed :)